src/entropies_estimators/nearest_neighbors/Goria.jl

```diff
@@ -0,0 +1,66 @@
+using StateSpaceSets: AbstractDataset, Dataset
+using Neighborhood: KDTree, NeighborNumber, Theiler
+using Neighborhood: bulksearch
+using SpecialFunctions: digamma
+
+export Goria
+
+"""
+    Goria <: DifferentialEntropyEstimator
+    Goria(k = 1, w = 0)
```

Comment:
```suggestion
    Goria(; k = 1, w = 0)
```
is what you mean right? without `;` it means optional positional arguments.